utilities/cluster_config_utils.py

```diff
@@ -38,12 +38,14 @@ def write(self, fp):
             fp.write("\n")
 
 
-def get_cluster(url, bucket_name):
+def get_cluster(url, bucket_name, get_bucket=False):
     timeout_options = ClusterTimeoutOptions(kv_timeout=timedelta(seconds=180), query_timeout=timedelta(seconds=300),
                                             config_total_timeout=timedelta(seconds=600))
     options = ClusterOptions(PasswordAuthenticator("Administrator", "password"), timeout_options=timeout_options)
     cluster = Cluster(url, options)
     cluster = cluster.bucket(bucket_name)
+    if get_bucket:
```

Comment:
Just curious: what is that for?

Reply:
wanted to access the cluster at the bucket level rather than the default collection, but I think I've found another way around it so this can be removed